modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -55,157 +60,261 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
 the changes from the codemod as a single isolated commit (separate from other changes) so you can
 roll back more easily if necessary.
 
-## Component Deprecations
+## Deprecations and Removals
+
+#### Deprecations
```

Comment:
should these go in the Glossary?

Reply:
I asked James this, but those are only relevant for one section vs main labs and preview are relevant across the upgrade guide

Reply:
Ok sounds good